nixos/modules/services/web-apps/pict-rs.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, pkgs, config, ... }:
+with lib;
+let
+  cfg = config.services.pict-rs;
+in
+{
+  options.services.pict-rs = {
+    enable = mkEnableOption "pict-rs server";
+    imageDirectory = mkOption {
```

Comment:
Personally I would call this `stateDir` or `dataDir` for consistency, but that's more opinion than anything. For this application the images are state.

Reply:
happy to change this. I have to say I was looking for a good name. I'm going to go for dataDir since it's a bit more expressive.